books/scss/detail.scss

```diff
@@ -1,3 +1,21 @@
+.back-list {
+  margin-top: 8vh;
```

Comment:
Avoid using `vh` for margins, it can bring visual bugs

Reply:
Have been improved